python/lsst/pipe/base/executionButlerBuilder.py

```diff
@@ -108,6 +109,11 @@ def _export(butler: Butler, collections: Optional[Iterable[str]], exports: Set[D
     exporter = RepoExportContext(butler.registry, butler.datastore, backend, directory=None, transfer=None)
     exporter.saveDatasets(exports)
 
+    # Need to ensure that the dimension records for outputs are
+    # transferred.
+    for _, dataIds in inserts.items():
+        exporter.saveDataIds(dataIds)
```

Comment:
I _think_ this should be fine, but we should be on the lookout for it running into the problem @SimonKrughoff reported [here](https://lsstc.slack.com/archives/C01FBUGM2CV/p1621983930190800) where we export visit records but not visit definition records.  I'm not sure exactly what he did differently.